core/services/job/spawner.go

```diff
@@ -107,14 +113,14 @@ func (js *spawner) RegisterDelegate(delegate Delegate) {
 
 func (js *spawner) runLoop() {
 	defer close(js.chDone)
+	defer js.destroy()
 
 	// Initialize the Postgres event listener for new jobs
-	var chNewJobs <-chan string
-	listener, err := js.orm.ListenForNewJobs()
+	newJobs, err := js.orm.ListenForNewJobs()
 	if err != nil {
-		logger.Errorw("Job spawner failed to subscribe to 'new job' events, falling back to polling", "error", err)
+		logger.Warn("Job spawner could not subscribe to new job events, falling back to polling")
```

Comment:
Do we really want to limp on in this instance or give up and error out the whole application?

Reply:
I wouldn't call it limping, as it's still fully functional, just a little slower.  Not erroring out the whole application might make whatever issue is causing this easier to debug.